### DISCUSSION TITLE
Skywalking and springboot integration,springboot version 2.3.12.RELEASE

### DISCUSSION THREAD

**Discussion by zhouyun0242** [apache/skywalking#9492] [Q&A]
After skywalking is integrated with springboot, the following errors will be reported when the project is started:
Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'org.springframework.context.annotation.internalAsyncAnnotationProcessor' defined in class path resource [org/springframework/scheduling/annotation/ProxyAsyncConfiguration.class]: Initialization of bean failed; nested exception is java.lang.ExceptionInInitializerError

**Comment by wu-sheng** [comment]
You didn't report the root cause. We can't know what it happened.
It is your responsibility to check the codes first.

**Reply by zhouyun0242** [reply]
thank you.
The problem is solved because the idea uses the short command line. It is OK to remove this configuration

**Reply by notdragonfly** [reply]
Hello, I also encountered this issue when using the probe jar package proxy in the Linux environment. Have you encountered it and how did you solve it?
